modules/alpha/moduletemplate-keda-v0.0.1-alpha.yaml

```diff
@@ -0,0 +1,63 @@
+apiVersion: operator.kyma-project.io/v1alpha1
+kind: ModuleTemplate
+metadata:
+  name: moduletemplate-keda
+  namespace: kcp-system
+  labels:
+    "operator.kyma-project.io/managed-by": "lifecycle-manager"
+    "operator.kyma-project.io/controller-name": "manifest"
+    "operator.kyma-project.io/module-name": "keda"
+  annotations:
+    "operator.kyma-project.io/module-version": "0.0.1-alpha"
+    "operator.kyma-project.io/module-provider": "internal"
+    "operator.kyma-project.io/descriptor-schema-version": "v2"
+spec:
+  target: remote
+  channel: alpha
+  data:
+    apiVersion: operator.kyma-project.io/v1alpha1
+    kind: Keda
+    metadata:
+      labels:
+        operator.kyma-project.io/kyma-name: keda-sample
+      name: keda-sample
+    spec:
+      logging:
+        operator:
+          level: "debug"
+  descriptor:
+    component:
+      componentReferences: []
+      name: kyma.project.io/module/keda
+      provider: internal
+      repositoryContexts:
+      - baseUrl: europe-west4-docker.pkg.dev/sap-se-cx-kyma-wookiee/dev-registry/unsigned
+        componentNameMapping: urlPath
+        type: ociRegistry
+      resources:
+      - access:
+          digest: sha256:e231a9b5dd60142699b323e9921bcad986aca29036af2f0b6ffe8a1747eba5e6
+          type: localOciBlob
+        name: keda
+        relation: local
+        type: helm-chart
+        version: 0.0.1-alpha
+      - access:
+          digest: sha256:f4a599c4310b0fe9133b67b72d9b15ee96b52a1872132528c83978239b5effef
+          type: localOciBlob
+        name: config
+        relation: local
+        type: yaml
+        version: 0.0.1-alpha
+      sources:
+      - access:
+          commit: 157f9933b29f107e9030b60411ee7f0e9a3bea7d
```

Comment:
commit cannot be verified, please double check it exists. I will have to base my review on main (8622f7f4eab5b64994a89eec5edd0f3e1082f8d6)

Reply:
fixed